src/app.js

```diff
@@ -1 +1,37 @@
 'use strict';
+
+const readline = require('readline');
+
+const { getRandomNumber } = require('./getRandomNumber');
+const { calculateBullsAndCows } = require('./calculateBullsAndCows');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+terminal.question('Guess a number: ', (number) => {
```

Comment:
i think will be better to create separate function with this logic, and just call it here

Reply:
agree